### DISCUSSION TITLE
What would be a good alternative to UTM for a Linux host

### DISCUSSION THREAD

**Discussion by phirestalker** [utmapp/UTM#6419] [General]
I will be moving away from mac. I don't like how hard it is to upgrade the hardware. I do like the M1 and all, but it comes at too high of a cost. Also, Apple refuses to treat these little minis as laptops and let me ship it in for repair. I live too far away from an Apple Store to make 2 trips.

Anyway, I want the ease of UTM for running windows, and a Whonix VM on Linux. I do not like VirtualBox at all. I like QEMU, but am open to other options. The solution needs to have easy graphical management of VMs, and excellent support for hardware acceleration (Hypervisor, and GPU).

Thanks all.

**Comment by irusensei** [comment]
Install libvirtd, qemu-kvm and virt-manager. The details might vary depending on distribution but I bet you can easily find guides.
